notebooks/concepts/imviz_roman_asdf.ipynb

```diff
@@ -321,7 +358,7 @@
    "outputs": [],
    "source": [
     "# Set the zoom level directly.\n",
-    "viewer.zoom_level = 1"
+    "viewer.zoom_level = 100"
```

Comment:
@kecnry , did you change this in your recent viewer zoom level PR?

Reply:
This is using the astrowidgets API, so no.  Plot options is now an alternative to set the radius instead.

Reply:
This is just a concept notebook though, so if the zoom-radius may change at all, then this is probably more stable in the meantime 🤷‍♂️

Reply:
As long as Roman users using this concept notebook understands that zoom level 100 isn't normal.

Reply:
Currently the only users who have access to Roman data products, or sufficient context to understand the contents of the files at those URLs, are on the Roman team. So I don't expect this will introduce much confusion. The primary function of this concept notebook is to serve as a usable checkpoint for the Roman Mission Office.